config/sink/placeholder.go

```diff
@@ -16,4 +16,4 @@ limitations under the License.
 
 // Package config is a placeholder that allows us to pull in config files
 // via go mod vendor.
-package config
+package sink
```

Comment:
Why don't we keep the `package config` like we did for `source` [here](https://github.com/knative-sandbox/eventing-redis/blob/06f8afa58df477399adeff03d268f738126635eb/config/source/placeholder.go)?

Reply:
@lionelvillard

Reply:
I ended up doing the opposite to be consistent with the other repos